### PR TITLE
added in margins to the header, aligned text to left

### PR DESCRIPTION
**Pull request by kathleenahall**
I hope this is what you were asking for!

### CODE REVIEW

**Comment by JoshuaVSherman** [comment]
@kathleenahall no the header section of the entire page where it says, `We celebrate God's grace and share His love in Christ!` is what I was asking for.  In addition, I was asking you to please align the text to the left on the Giving page top section (which it looks like you did), but not sure about the margins now, you should definitely test all of this out on various device sizes, thanks

**Comment by kathleenahall** [comment]
I'm still a bit confused. I don't see any distinguishable differences
between the header on the website vs. the header opening from Facebook so
I'm not sure what I'm trying to fix.

**Review by JoshuaVSherman — CHANGES_REQUESTED** [review]
@kathleenahall needs a rebase and fix any merge conflicts please

**Comment by JoshuaVSherman** [comment]
@kathleenahall you have to resolve the merge conflicts please

**Review by JoshuaVSherman — CHANGES_REQUESTED** [review]
@kathleenahall there are merge conflicts that must be resolved before we can merge your branch please

File: src/containers/HabitatProject/HabitatProjectContent.tsx [hunk 1]
```diff
@@ -50,6 +50,107 @@ const Responsibilities = (props: { children: any; }) => {
   );
 };
 
+
+const AboutProject = () => {
```
Comment (JoshuaVSherman) [inline src/containers/HabitatProject/HabitatProjectContent.tsx:54]: @kathleenahall I don't think you meant to make these changes on this page, please put it back how it was before. I think you messed up when you were fixing merge conflicts, thank you

**Review by JoshuaVSherman — CHANGES_REQUESTED** [review]

File: src/containers/HabitatProject/HabitatProjectContent.tsx (hunk 1 above)
Comment (kathleenahall) [reply src/containers/HabitatProject/HabitatProjectContent.tsx:54]: This does look like something that happened when I tried to fix the merge conflicts in Github, but these changes are not showing up in vs code so I'll push everything up once I fix the merge conflicts in vscode.

**Pull request closed without merging**